cypress/integration/add-delete-collection.ts

```diff
@@ -0,0 +1,36 @@
+before(() => {
+  cy.clearCookies()
+  cy.clearLocalStorage()
+})
+
+it('can add cards to the collection', () => {
+  cy.visit('/')
+  cy.findAllByRole('link', { name: 'Search' }).click()
```

Comment:
your finding all links with the name role, but which one are you clicking? findAllBy* is going to return an array, you need to specify which one you want to click.